### PR TITLE
chore(3000): Remove organization from breadcrumbs

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

I propose we introduce the following split in 3000: organization switching lives in the account popover, while project switching lives in breadcrumbs. Currently you can do the former from breadcrumbs AND the popover, while the latter ONLY from breadcrumbs. I'm not sure that this is entirely good for the discoverability of projects, but I definitely think that it's redundant for orgs.

To this end, this PR removes the organization breadcrumb in 3000.

### CODE REVIEW

File: frontend/__snapshots__/replay-listings--recordings-play-lists.png
Comment (daibhin) [inline frontend/__snapshots__/replay-listings--recordings-play-lists.png:1]: Looks like the pre-3000 UI is also affected when the code suggests that it shouldn't be?

**Comment by thmsobrmlr** [comment]
I initially thought this is a great idea - people likely don't switch the organization much and this simplifies the UI. Unexpectedly the org switcher seems to be used quite a bunch by users though https://app.posthog.com/data-management/actions/59306 (given I've not made a mistake setting up the action).

**Comment by Twixes** [comment]
Hmm, that's interesting. Yeah, it does look like the org switcher has a bit over 1/3 of the project switcher's DAUs in 3000 – which is a big chunk, because project switching is not even available elsewhere. Though also interesting, because I would suspect more users to be multi-project rather than multi-org.
New organizations are cread from breadcrumbs more than from the account popover.
([Insight for both analyses.](https://app.posthog.com/insights/D1EOmY1w))
So let's say we keep the organization in breadcrumbs.

**Comment by corywatilo** [comment]
I've actually been using the breadcrumb a ton to switch orgs! Ultimately I think it should move to the nav column (likely at the top) as this turned out to be a pretty good solution for a company that rhymes with Leap. But what we have works for now and definitely helps keep all the controls in one place and prevents thrash of what to expect where.

Now if only we can clear up the ambiguity between project settings, personal settings, and billing...

**Pull request closed without merging**